test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,34 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
+        // empty
+        assertNotNull();
+
+        // only one object
+        assertNull((Object) null);
+        assertNotNull(1);
+        assertNotNull("");
+        assertNotNull("abc");
+
+        // all null
+        assertNull(null, null, null);
+
+        // some null
+        assertNull(1, null);
+        assertNull("abc", null, "");
+        assertNull(null, Integer.valueOf(1));
+    }
+
+    private void assertNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
+    }
+
+    private void assertNotNull(Object... objects) {
```

Comment:
As mentioned in tutorial, there is no need to abstract out `assertTrue` and `assertFalse` as separate methods since they are rather short. Existing tests do that to avoid repeating `Arrays.asList` everywhere. You can use them directly in `isAnyNull`.